website/docs/features/cors.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: cors
+title: CORS with Yoga
+sidebar_label: CORS
+---
+
+Yoga comes with CORS (Cross origin resource sharing) support out of the box - CORS can be configured when creating the server either by passing a CORSOptions object, or a builder function that returns the CORSOptions object.
```

Comment:
Maybe it makes sense to add 1 or 2 sentences on what CORS is and why it is important?